scripts/lint_translations.clj

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bb
+
+(ns lint-translations
+  (:require [babashka.pods :as pods]))
+
+(pods/load-pod 'clj-kondo/clj-kondo "2023.09.07")
+(require '[pod.borkdude.clj-kondo :as kondo])
+(require '[cheshire.core :as json])
+(require '[clojure.set :as set])
+
+(def src-paths ["src"])
+
+(def translation-file "translations/en.json")
+
+;; set the following to true when solving https://github.com/status-im/status-mobile/issues/17811
+(def flag-show-non-namespaced-translation-keys false)
+(def flag-show-non-namespaced-translation-keys-occurrences false) ;; this makes the output super verbose!
+
+;; set the following to true when solving https://github.com/status-im/status-mobile/issues/17813
+;; and keep it permanently on after #17811 and #17813 have both been solved
+(def flag-show-unused-translation-keys false)
+
+(def flag-show-missing-translation-keys true)
+
+(defn- safe-name
+  [x]
+  (when x (name x)))
+
+(defn- ->keyword
+  [analysis-keyword]
+  (keyword (safe-name (:ns analysis-keyword)) (:name analysis-keyword)))
+
+(defn- report-issues
+  [incorrect-usages]
+  (doseq [incorrect-usage incorrect-usages]
+    (->> incorrect-usage
+         ((juxt :filename :row :reason ->keyword))
+         (apply format "%s:%s %s %s")
+         println)))
+
+(defn- extract-translation-keys
+  [file]
+  (-> file slurp json/parse-string keys))
+
+
+(def ^:private probably-unused-warning
+  (format "Probably unused translation key in %s:" translation-file))
+
+(defn -main
+  [& _args]
+  (let [result                           (kondo/run!
```

Comment:
It would be nice to print a message to the dev because the linting takes a while, maybe something like `Linting translations...`